README.md

```diff
@@ -268,17 +270,15 @@ docker tag pravega/pravega-operator [REGISTRY_HOST]:[REGISTRY_PORT]/pravega/prav
 docker push [REGISTRY_HOST]:[REGISTRY_PORT]/pravega/pravega-operator
 ```
 
-where:
```

Comment:
Not a note. Please revert change.

Reply:
@adrianmo 
i will add the below mentioned content and create TOC 👍